### PR TITLE
Include cdap-common test-jar in unit-test to bring in logback-test.xml

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: cdap-operational-stats-core/pom.xml [hunk 1]
```diff
@@ -37,7 +37,13 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-yarn-client</artifactId>
-      <version>${hadoop.version}</version>
```
Comment (chtyim) [inline cdap-operational-stats-core/pom.xml:40]: This should comes from the parent pom dependency management.

Comment (awholegunch) [inline cdap-operational-stats-core/pom.xml:40]: Shouldn't this line stay here?

Comment (chtyim) [reply cdap-operational-stats-core/pom.xml:40]: See my comment.

**Comment by bdmogal** [comment]
Lgtm

**Comment by chtyim** [comment]
Build passed http://builds.cask.co/browse/CDAP-DUT5133

**Pull request merged**